srv/modules/runners/bootstrap.py

```diff
@@ -76,11 +66,10 @@ def ceph(non_interactive=False):
             pager(_read_policy_cfg())
             if not prompt("Do you want to continue?"):
                 return 'aborted'
```

Comment:
Right, this is something we'll probably change in the future. 

Right now I'm (mostly) returning bools to react on the returns in other functions. However, it's probably not the most user friendly way to _print_ booleans to the screen. So we might come up with something else. Here are some ideas:

1) always return tuples `return_code<bool>, error/sucess_message<str>`
2) Interprete the return<bool> based on the context the runner is invoked (human vs machine)
    A machine would get the boolean variant, while the Human get's a translated 'success' or 'failure'

I think I prefer option 1) since this allows us to tell the user what _exactly_ went wrong.

Reply:
I prefer option 1) because it's much more explicit

Reply:
:+1: for option 1